src/index.js

```diff
@@ -1,7 +1,7 @@
 const {
-  KeyManager,
   deriveSeedFrom
-} = require("@holo-host/wasm-key-manager");
+} = require("./deriveSeedFrom")
+const KeyManager = require("./keyManager")
```

Comment:
I thought we decided not to have this implementation in the lib.

Reply:
I was just keeping the half completed js implementations around. But I'll move them to a branch